blue-xdma/src/include/Config.bsv

```diff
@@ -0,0 +1,70 @@
+package Config;
+
+import BlueAXI :: *;
+
+typedef 3000 WAITRESET;
+typedef 10000 STOPAFTER;
+
+typedef 'h80 TESTLENGTH;
```

Comment:
use decimal for `TESTLENGTH` instead of hex, for easy understanding,

for `TESTSRCADDR` and `TESTDSTADDR`, it's OK to use hex

Reply:
The hexadecimal here was adjusted to accommodate the test in the example design, and has been modified in the latest commit.